ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/DefaultSynchronizer.java

```diff
@@ -99,6 +99,34 @@ public DefaultSynchronizer(
             syncState,
             metricsSystem,
             terminationCondition);
+
+    // TODO add this code when snapsync will be ready
+    /*if (SyncMode.X_SNAP.equals(syncConfig.getSyncMode())) {
+      this.fastSyncDownloader =
+          SnapDownloaderFactory.createSnapDownloader(
+              syncConfig,
+              dataDirectory,
+              protocolSchedule,
+              protocolContext,
+              metricsSystem,
+              ethContext,
+              worldStateStorage,
+              syncState,
+              clock);
+    } else {
+      this.fastSyncDownloader =
+          FastDownloaderFactory.create(
+              syncConfig,
+              dataDirectory,
+              protocolSchedule,
+              protocolContext,
+              metricsSystem,
+              ethContext,
+              worldStateStorage,
+              syncState,
+              clock);
+    }*/
```

Comment:
can we omit this from the PR?

Reply:
I prefer to omit this one and activate this part of the code when all the PRs will be merged. In two PRs maybe not more